react-events-and-state/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(state => ({ isClicked: true }));
```

Comment:
For most cases you don't need to pass `setState` a callback function, you can just pass it an object with the properties you want to update.

```JSX
this.setState({ isClicked: true });
```

Reply:
I got it! Thanks!